### DISCUSSION TITLE
Add delete account option

### DISCUSSION THREAD

**Discussion by anmarchenko** [dailydotdev/daily#1075] [Feature Requests]
I would like to delete my account but I couldn’t find this option in profile settings. There is also no support contacts so it is not clear how to delete account completely.

**Comment by nimrodkra** [comment]
Here's a guide:
https://docs.daily.dev/docs/your-profile/deleting-your-profile


On Tue, Feb 13, 2024 at 11:23 AM, Andrey Marchenko ***@***.***

**Reply by anmarchenko** [reply]
thank you!

**Comment by pedrodovale** [comment]
If you're like me and the accepted solution does not work for you:

The UI has changed slightly since then. Now, to delete your account, go to:

1. Settings
2. Account
3. Danger Zone
4. Delete account

or simply click this click: https://app.daily.dev/settings/security

I would also recommend updating the support documentation: https://docs.daily.dev/docs/your-profile/deleting-your-profile @nimrodkra

**Reply by idoshamun** [reply]
@pedrodovale thanks for the reminder. we fixed the docs
